src/utils/permission.tsx

```diff
@@ -0,0 +1,15 @@
+import { Linking, Alert } from "react-native";
+
+export const requestPermissions = () => {
```

Comment:
nit: If this is just a one-off I'd be okay bundling it with the hook code that calls it. Unless you foresee us adding a bunch more code here for permissions.